astropy/table/table.py

```diff
@@ -2153,6 +2153,57 @@ def rename_column(self, name, new_name):
 
         self.columns[name].info.name = new_name
 
+    def rename_columns(self, names, new_names):
+        '''
+        Renames multiple columns.
+
+        Parameters
+        ----------
+        names: list
+            A list of existing column names.
+        new_names: list
+            A list of new column names.
+
+        Examples
+        --------
+        Create a table with three columns 'a','b','c'::
+
+            >>> t = Table([[1,2],[3,4],[5,6]], names=('a','b','c'))
+            >>> print(t)
+              a   b   c
+             --- --- ---
+              1   3   5
+              2   4   6
+
+        Renaming columns 'a' to 'aa' and 'b' to 'bb'::
+            >>> names = ('a','b')
+            >>> new_names = ('aa','bb')
+            >>> t.rename_columns(names, new_names)
+            >>> print(t)
+             aa  bb   c
+            --- --- ---
+              1   3   5
+              2   4   6
+        '''
+
+        if self._is_list_or_tuple_of_str(names):
+            names = names
+
+        if self._is_list_or_tuple_of_str(new_names):
+            new_names = new_names
+
+        if isinstance(names, str):
```

Comment:
Delete this block and the one below.  The API does not specify that one can provide a string instead of a list/tuple of strings, and there is no reason to allow this case since the existing `rename_column` method handles that.